docs/moleculenet.rst

```diff
@@ -1,31 +1,35 @@
+The MoleculeNet suite has curated a whole range of datasets and loaded them into DeepChem
+:code:`dc.data.Dataset` objects for convenience.
 
 Contributing a new dataset to MoleculeNet
 -----------------------------------------
 
-If you are proposing a new dataset to be included in the MoleculeNet benchmarking suite, 
-please follow the instructions below. Please review the `datasets already available in MolNet <http://moleculenet.ai/datasets-1>`_ before contributing.
+If you are proposing a new dataset to be included in the 
+MoleculeNet benchmarking suite, please follow the instructions below. 
+Please review the `datasets already available in MolNet`_ before contributing.
 
-0. Read the `Contribution guidelines <https://github.com/deepchem/deepchem/blob/master/CONTRIBUTING.md>`_.
+0. Read the `Contribution guidelines`_.
 
-1. Open an `issue <https://github.com/deepchem/deepchem/issues>`_ to discuss the dataset you want to add to MolNet.
+1. Open an `issue`_ to discuss the dataset you want to add to MolNet.
 
-2. Implement a function in the `deepchem.molnet.load_function <https://github.com/deepchem/deepchem/tree/master/deepchem/molnet/load_function>`_ module following the template function `deepchem.molnet.load_function.load_mydataset <https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/load_function/load_mydataset.py>`_. Specify which featurizers, transformers, and splitters (available from `deepchem.molnet.defaults <https://github.com/deepchem/deepchem/tree/master/deepchem/molnet/defaults.py>`_) are supported for your dataset. 
+2. Implement a function in the `deepchem.molnet.load_function`_ 
+   module following the template function `deepchem.molnet.load_function.load_dataset_template`_.
+   Specify which featurizers, transformers, and splitters (available from
+   `deepchem.molnet.defaults`_) are supported for your dataset. 
 
-3. Add your load function to `deepchem.molnet.__init__.py <https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/__init__.py>`_ for easy importing.
+3. Add your load function to `deepchem.molnet.__init__.py`_ for easy importing.
 
 4. Prepare your dataset as a .tar.gz or .zip file. Accepted filetypes include CSV, JSON, and SDF.
 
-5. Ask a member of the technical steering committee to add your .tar.gz or .zip file to the DeepChem AWS bucket. Modify your load function to pull down the dataset from AWS.
+5. Ask a member of the technical steering committee to add your .tar.gz or .zip file 
+   to the DeepChem AWS bucket. Modify your load function to pull down the dataset from AWS.
 
-6. Submit a [WIP] PR (Work in progress pull request) following the PR `template <https://github.com/deepchem/deepchem/blob/master/docs/molnet_pr_template.md>`_.  
+6. Submit a [WIP] PR (Work in progress pull request) following the PR `template`_.
 
-Load Dataset Template
----------------------
-
-.. autofunction:: deepchem.molnet.load_function.load_dataset_template.load_mydataset
```

Comment:
This line broke the docs build. `load_dataset_template` raise error when being loaded.